core_backend/app/contents/models.py

```diff
@@ -133,6 +153,10 @@ async def delete_content_from_db(
     """
     Deletes a content from the database
     """
+    association_stmt = delete(content_tags_table).where(
```

Comment:
Needs to delete the content_tags association before deleting from db

Reply:
Nice!